Changelog.md

```diff
@@ -1,9 +1,14 @@
 # Monoidal containers
 
+# 0.5.0.0
+
+  * Make `fromList`, `insert`, and `mapKeys` from `Data.Map.Monoidal` and `Data.Map.Monoidal.Strict` require `Semigroup` on values to properly capture monoidal behavior instead of reverting to the left-biased semantics of `Data.Map`.
+  * Support `these` 0.8.0
```

Comment:
fyi, there's `these-1` (and `semialign-1`) too now.

Reply:
Whew...well I guess we'll have to open another PR but at this rate we don't have to worry about those versions any time soon :joy:.

Reply:
@bgamari If you'd like help maintaining this project, I'd be happy to help. We use this library extensively.